src/common/copilot/telemetry/ITelemetry.ts

```diff
@@ -15,4 +15,7 @@ export interface IProDevCopilotTelemetryData {
     feedbackType?: string,
     responseStatus?: number,
     codeLineCount?: number,
+    geoName?: string,
+    aibEndpoint?: string,
```

Comment:
Why do we need to send AI Builder endpoint to telemetry?

Reply:
Is it going to the correct aibEndpoint and which island it is going we need for debugging.

Reply:
Can this be part of log files instead?

Reply:
Is there any specific reason you think this is a problem? If we put this in log file then we are dependent on Customer to share this for debugging.